e2e-tests/README.md

```diff
@@ -57,6 +57,10 @@ Additionally, it includes values for handling timeouts and retries: `pageLoadTim
 
 ⚠️ The `pageLoadTimeout` was added because deal submissions can sometimes take time, especially if there are many external API calls involved. Eventually, these API calls will be optimized to reduce submission time and make the E2E tests run more efficiently.
 
+### Importing from `@ukef/dtfs2-common`
+
+Some test fixtures which are used by both `utils/mock-data-loader` and `e2e-tests` are defined in the `libs/common` workspace (see [portal users](../libs/common/src/test-helpers/mock-data/portal-users.mock.ts), for example). However, we cannot import these modules directly in cypress files which are served to the cypress client as the import requires a `node` environment, whereas cypress runs in-browser. To overcome this, constants imported from `libs/common` should be stored as cypress environment variables which can be accessed via the `Cypress.env` command. These env variables are defined in [`e2e-tests/support/cypress-env.ts`](./support/cypress-env.ts), with a `getCypressEnvVariable` helper function and `CYPRESS_ENV_KEY` constant defined in [e2e-fixtures](./e2e-fixtures/). An example implementation of the helper function and constant can be seen in [the portal users fixture](./e2e-fixtures//portal-users.fixture.js). The types have also been defined in such a way that you will get a `tsc` failure if you add a new environment variable to [`e2e-tests/support/cypress-env.ts`](./support/cypress-env.ts) without updating the `CYPRESS_ENV_KEY` constant.
+
```

Comment:
Hmm I can see how this works but I'm not completely sold on it, it feels a little hacky to highjack the environment variables in this way, but I am not very familiar with cypress. I wonder if there is a way to import these mocks within the client side tests from the common library without loading anything that depends on mongo. I have no idea if it would work but have we tried importing the mocks with `@ukef/dtfs2-common/path/to/folder` where the folder doesn't import from mongo at all?

Reply:
If we wanted to add a specific import via `@ukef/dtfs2-common/<path>`, we need to modify the `package.json` in `libs/common` - we can't just do `@ukef/dtfs2-common/test-helpers`, for instance. However, we do import `mongodb` in the `MOCK_USERS` definition indirectly as it imports `MOCK_BANKS` which uses the `OptionalId` utility type from `mongodb`, so we can't really avoid using that import 😢 

@ChristianMcc would you possibly be able to provide any ideas linked to this? Ruby is correct that this solution is a bit hacky, but I can't think of any other non-hacky solutions to get around this :/

Reply:
I wasn't aware there was a directly dependency on mongo here. As discussed in person, I think we simply shouldn't be typing the mock banks with the mongo type and generally I don't think the client side of end to end tests actually need to have access to anything that is mongo related.

Reply:
To conclude this discussion, I have moved the mock data back to `utils/mock-data-loader` as they shouldn't really be defined in `libs/common` - this means this `e2e-tests` issue no longer applies